blocks/date-block/features/date/default.jsx

```diff
@@ -38,11 +33,11 @@ class ArticleDate extends Component {
 
   render() {
     const { displayDate } = this.state;
-    const { arcSite, classNames } = this.props;
+    const { classNames } = this.props;
     return (
-      <StyledTime key={displayDate} className={`date ${classNames}`} dateTime={displayDate} primaryFont={getThemeStyle(arcSite)['primary-font-family']}>
+      <PrimaryFont as="time" key={displayDate} className={`${classNames}`} dateTime={displayDate}>
```

Comment:
looks like the classname of date is removed? is that deliberate?

Reply:
i was testing something and forgot to put it back, thanks for catching that